install/kubernetes/helm/istio/values-istio-all.yaml

```diff
@@ -0,0 +1,37 @@
+# This is used to generate istio.yaml
```

Comment:
Not sure what this file does - but normal practice is to just override.

There is a values.yaml that has defaults - and overrides should only replace specific values.

Reply:
This is to generate all components by setting all options to be true. I am only moving some Makefile logic to Helm.

Reply:
@yusuoh the advantage only overriding what is a delta from `values.yaml` is less maintenance work keeping the default configs in sync with other configs used for test cases.

`rbacEnabled=true` in master `values.yaml`
`ingress.enabled=true` in master `values.yaml`
`refreshInterval=1" in master `values.yaml`

istiotesting should only be used to generate test manifests (mentioned @manderjog in a different review) and should not be overridden here (and the value overridden is already false so it would be redundant).

refreshInterval is also defaulted to 1 second in values.yaml.